### ISSUE TITLE
Windows builds crashing on startup with "No Primary Display Defined"

### ISSUE THREAD

**Issue by zshoals** [Kode/Kinc#552]
This appears to have been introduced with this commit: https://github.com/Kode/Kinc/commit/7a8365b9836a3f1c69a166b898f0d558f92bf096

The build target was a full build for Windows D3D11, if that's relevant.

Reverting to a commit before this point causes everything to work as expected.

**Comment by RobDangerous** [comment]
I can not reproduce that. Do you do any Kinc things before calling kinc_init?

**Comment by RobDangerous** [comment]
Aha, it happens with Kha applications!

**Comment by zshoals** [comment]
I don't believe so. I should have specified already (my bad honestly), but this happens during the compilation of a Kha project. I have not made any adjustments to Kinc or anything directly. 

Lol, timing :).

**Comment by zshoals** [comment]
Sorry for not specifying that earlier, my bad.
